Dev21

SleeplessShu opened the pull request:
привет. хорошего тебе дня и приятных событий)

SleeplessShu committed:
Спринт 21 коммит 00

SleeplessShu committed:
Спринт 21 коммит 01

добавлена база данных избранных треков
в плеере нажатие на лайк добавляет или удаляет трек из избранного
в медиатеке треки отражаются от новых к старым или заглушка если бд пуста

kosstyango commented:
⚠️ Критические замечания:
Треки в списке "Избранное" отображаются НЕ в обратном (и не в прямом) порядке.

https://github.com/user-attachments/assets/e1372931-b75c-4503-b942-32be8456d40e


🍏 Рекомендации, не влияющие на результат проверки работы:
Треки выглядят неряшливо, т.к. длительность трека отображается на случайном расстоянии от названия альбома, а название альбома иногда необоснованно обрезается до двух(!) букв, хотя место в строке еще есть... (См. скрин) 
![Спринт_21_Бурмистров](https://github.com/user-attachments/assets/0f1b67ef-462d-4ede-a2e4-39e7726700d7)

SleeplessShu committed:
Спринт 21 коммит 02

исправлен порядок отображения треков в медиатеке
исправлена отображаемая длина текста в имени исполнителя
